code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -67,8 +67,15 @@
 	display_name = "maid neckpiece"
 	path = /obj/item/clothing/neck/maid
 
-/datum/gear/accessory/gloves
-	subtype_path = /datum/gear/accessory/gloves
```

Comment:
Glove subtype definition for loading all glove subtypes, including the current evening gloves, is it not?

Reply:
It's there to skip the subtype during loading given it doesn't contain anything and is just there for organization, indeed.

Reply:
As in, it skips /datum/gear/accessory/gloves and only loads it's subtypes such as /datum/gear/accessory/gloves/evening